### ISSUE TITLE
Yippiiiaaaah

### ISSUE THREAD

**Issue by iRove108** [jeffreyzpan/adversarial-playground#1]
This repo is very insightful and I have learned much from it. Now off to eat my pizza!

C ya

**Comment by createandbuild** [comment]
I completely agree. A very jigglry job! I especially enjoyed reviewing your use of the MIT License.

**Comment by iRove108** [comment]
Yes, we would never dare to be an adversary to the MIT license

**Comment by jeffreyzpan** [comment]
yippy yippee yaha yah jigllryryryry!!!!
